public/src/components/channelManagement/variantEditorCtaFieldsEditor.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect } from 'react';
```

Comment:
This was factored out of the `VariantEditorCtaEditor` so that it can be shared by the new `VariantEditorSecondaryCtaEditor`